test/CMakeLists.txt

```diff
@@ -49,6 +49,14 @@ if(CHECK_FOUND)
 
 endif(CHECK_FOUND)
 
+
```

Comment:
I installed check library, but I am the newbie on linux development and git-hub, I just want all the modifications can be arranged together.
If something important I ignore, please point to me.
I will appreciate to you.